mnist/README.md

```diff
@@ -6,165 +6,96 @@ This example guides you through the process of taking an example model, modifyin
 
 Before we get started there a few requirements.
 
-### Kubernetes Cluster Environment
+## 1.Kubernetes Cluster Environment
```

Comment:
The numerical formatting makes the document difficult to read. It now looks like every step in the guide is a prerequisite.

Reply:
this numerical format is very helpful for beginners, they just need to follow the steps one by one and finally get a good result.

Any suggestions on this? @jlewi

Reply:
Can we just make installing the preqruisites step #1? 

Also can you add a space between the "." and the text.